### DISCUSSION TITLE
How to disable Ideavim just for pop-up windows?

### DISCUSSION THREAD

**Discussion by Zackhardtoname** [JetBrains/ideavim#825] [Q&A]
I thought I have seen a way to do set that, but I couldn't find it.
The problem is rewarding the pop-up windows for GitLab review mode.

Steps to reproduce
1. Enable ideavim in latest version of CLion
2. Clone a repo
3. Open the GitLab tab
4. Select a new merge request.
5. Click on the edit icon to enter the file
6. Hover over any line to the left and click on the "+" icon
7. Type some characters
8. Delete some characters
9. Type some more
10. Delete some more
11. Note that characters are not actually deleted from the comment, but the line of code
12. Disable Ideavim
13. Try typing and deleting again, which works great now.

**Comment by citizenmatt** [comment]
Not sure what you mean by pop up windows. What are you trying to disable and where?

**Reply by Zackhardtoname** [reply]
I just added the steps to reproduce the problem. Thank you!
